### PR TITLE
Migration proposition to cozy-konnector-libs v4

### PR DESCRIPTION
**Pull request by doubleface**

### CODE REVIEW

**Comment by doubleface** [comment]
@nicofrand It should be better now

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+            .replace(" €", "")
+            .replace(",", ".");
+        const status = cells
+            .eq(4)
+            .text()
+            .trim()
+            .toLowerCase();
+
+        // Hacky to way to check without dealing with accents
+
```
Comment (nicofrand) [inline src/index.js]: Please remove this blank line

**Comment by clochix** [comment]
Ping @doubleface 

FTR, Nicofrand has been notified of a coming soon refactoring of this website so this connector will probably need some love in the next few weeks.

File: package.json [hunk 2]
```diff
@@ -9,37 +9,44 @@
...
     "license": "AGPL-3.0",
-    "main": "./index.js",
-    "dependencies": {
-        "cheerio": "0.22.0",
-        "cozy-konnector-libs": "2.4.1",
-        "request": "2.81.0"
+    "main": "./src/index.js",
+    "eslintConfig": {
+        "extends": [
+            "eslint-config-cozy-app"
```
[the remark this answers is not in the document]
Comment (doubleface) [reply package.json]: Yes, but you have to keep the "main" sectionYes, but you have to keep the "main" section

**Pull request merged**